### ISSUE TITLE
Cheat Engine 7.4 and certain PC Games...

### ISSUE THREAD

**Issue by chaosjs0010** [cheat-engine/cheat-engine#2282]
Hello there,  On Cheat Engine 7.4 I noticed that on certain PC Games that Module Addresses don't work correctly and BTW It does not matter if the Operating System is WIndows 11 or Windows 10 x64 I still will get the exact same problem on Certain PC Games!  Question is Is it fixable or Do I have to wait until the next version of Cheat Engine possibly? I have 64Gb of RAM and currently running Windows 11 Professional with all Windows Updates Applied including Optional Windows Updates also!  my CPU is an AMD Ryzen 9 5900X and my Motherboard is an MSI MAG B550 TOMAHAWK as well..

**Comment by cheat-engine** [comment]
Define not work correctly.
Do you mean that the base address of a module is missing or just wrong?

Is the target .NET/Mono? If so, you'll have to use the mono features/.net symbols instead. They do not have module addresses

**Comment by chaosjs0010** [comment]
I actually meant that the module addressing for the game exe appears to be say for example for game.exe it will display something like 7FFF10000000 or cloese to that on the top for the base addressing for the module address in question and on the bottom 7FFFFFFFFFFFF or close to it or something like that instead of what it suppose to look like!  Anyways, Hope that helps you understand a bit better what I meant.

**Comment by chaosjs0010** [comment]
For example Aegis Descent is one PC Game I have been unable to get the module addressing to appear correctly for.

**Comment by cheat-engine** [comment]
Go to settings->extra and disable all options there. 
Does it work then?

e.g this is my view when having Casper-Win64-Shipping.exe open in CE:
![image](https://user-images.githubusercontent.com/12006555/180815105-ab92ea4c-9505-4063-ac67-acdc5c5022cd.png)

**Comment by chaosjs0010** [comment]
No, It still does not work afterwards and just so you know I have the settings extras all unchecked right now anyways. and like I said it still does not work regardless.

**Comment by cheat-engine** [comment]
Besides anti cheat which I don't see in this game, it might be an anti virus that has hooked some API's and bugs out on CE

When you do an aob scan for :"48 83 EC 28 48 85 C9 74 11 83 FA 09 7D 11 85 D2" (in all memory, executable and copy-on-write greyed/enabled) what exact address do you get ?

**Comment by chaosjs0010** [comment]
For the AOB Scan I get this address 7FF6DB0E0E90 instead and by the way I also noticed that cheat engine 7.4 does not detect the address in question as a module address for some reason and I don't know why that is on my machine either. So, I am at a loss to figure it out as well..
